layouts/index.html

```diff
@@ -29,17 +29,15 @@
 
 				{{ .Content }}
 			{{ end }}
-			
```

Comment:
Apologies, my editor is set up to do this.  Let me know if it's a problem and I can undo this.